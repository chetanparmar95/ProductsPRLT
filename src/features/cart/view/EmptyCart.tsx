import React from 'react';
import { Text, StyleSheet, View } from 'react-native';

const EmptyCart = () => {
  return (
    <View style={styles.emptyCartContainer}>
        <Text style={styles.emptyText}>Your cart is empty</Text>
        <Text style={styles.descriptionText}>Start shopping to add items to your cart.</Text>
    </View>
  );
};

export default EmptyCart;

const styles = StyleSheet.create({
    emptyCartContainer: {
        flex: 1,
        alignItems: 'center',
        justifyContent: 'center',
    },
    emptyText: {
        fontSize: 24,
        fontWeight: 'bold',
        marginBottom: 16,
    },
    descriptionText: {
        fontSize: 18,
        color: '#888',
        textAlign: 'center',
    },
});