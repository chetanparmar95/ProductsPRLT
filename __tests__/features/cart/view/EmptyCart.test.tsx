
import React from 'react';
import { render } from '@testing-library/react-native';
import EmptyCart from '../../../../src/features/cart/view/EmptyCart';

describe('EmptyCart Component', () => {
  it('EmptyCart renders correctly', () => {
    const { toJSON } = render(<EmptyCart />);
    expect(toJSON()).toMatchSnapshot();
  });
})
