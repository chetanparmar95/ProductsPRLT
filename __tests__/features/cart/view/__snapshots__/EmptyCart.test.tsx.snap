// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyCart Component EmptyCart renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      {
        "fontSize": 24,
        "fontWeight": "bold",
        "marginBottom": 16,
      }
    }
  >
    Your cart is empty
  </Text>
  <Text
    style={
      {
        "color": "#888",
        "fontSize": 18,
        "textAlign": "center",
      }
    }
  >
    Start shopping to add items to your cart.
  </Text>
</View>
`;
